apm-sniffer/apm-sdk-plugin/jedis-plugins/jedis-2.x-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jedis/v3/JedisMethodInterceptor.java

```diff
@@ -37,7 +39,13 @@ public class JedisMethodInterceptor implements InstanceMethodsAroundInterceptor
     @Override
     public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allArguments, Class<?>[] argumentsTypes,
                              MethodInterceptResult result) throws Throwable {
-        String peer = String.valueOf(objInst.getSkyWalkingDynamicField());
+        String peer;
+        if (objInst instanceof Jedis) {
+            Client client = ((Jedis) objInst).getClient();
+            peer = client.getHost() + ":" + client.getPort();
+        } else {
+            peer = String.valueOf(objInst.getSkyWalkingDynamicField());
+        }
```

Comment:
Do you have to do through this way? This way is much slower in the runtime.
This interceptor was only executing codes relative to this redis client, but right now, it has to build peer again and again.
Considering accessing redis is super fast, this could be sensitive.
What do you think?

Reply:
get it，the peer can build in consturctor just when init，and set it to SkyWalkingDynamicField